front/src/components/HeaderMenu/HeaderMenuContainer.tsx

```diff
@@ -149,13 +149,13 @@ function HeaderMenuContainer() {
           },
           {
             link: `${oldAppUrlWithBase}&action=products`,
-            name: "Manage Products",
+            name: "Classic Manage Products",
             requiredAbps: ["manage_products"],
             external: true,
           },
           {
             link: `/bases/${baseId}/products`,
-            name: "ASSORT Products",
+            name: "Manage Products",
```

Comment:
@HaGuesto just to make sure:
- all users with beta-level >=4 (and manage_products ABP) can access the v2 ManageProductsk
- any user with manage_products ABP can access Classic Manage Products in dropapp, and there they see the "old" list of custom products (some of which might be SP instantiations) without tabs

Reply:
yes to both, but feel free to test